docs/source/common/lightning_module.rst

```diff
@@ -1474,18 +1470,6 @@ on_train_batch_end
 .. automethod:: pytorch_lightning.core.lightning.LightningModule.on_train_batch_end
     :noindex:
 
-on_epoch_start
-~~~~~~~~~~~~~~
-
-.. automethod:: pytorch_lightning.core.lightning.LightningModule.on_epoch_start
-    :noindex:
-
-on_epoch_end
-~~~~~~~~~~~~
-
-.. automethod:: pytorch_lightning.core.lightning.LightningModule.on_epoch_end
-    :noindex:
-
```

Comment:
do these need to be removed? wont the deprecation warning show up on the docs site?

should we move deprecated hooks to the bottom of the file so they're still found but not as discoverable?

Reply:
it will be visible if someone goes under API references or search for it. Here we are explicitly mentioning them. Since we are deprecating them, it just helps reduce the visibility since it's no longer recommended, and users shouldn't care for it.